src/sentry/integrations/github/client.py

```diff
@@ -103,6 +103,19 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> JSONData:
 
         return tree
 
+    def get_trees_for_org(self, org_name: str) -> JSONData:
+        """
+        This fetches tree representations of all repos for an org.
+        """
+        trees: JSONData = {}
+        repositories = self.get_repositories()
+        # XXX: In order to speed up this function we will need to parallelize this
+        # Use ThreadPoolExecutor; see src/sentry/utils/snuba.py#L358
```

Comment:
I have filed tasks to address the parallelization and test it. For now we can go ahead and use the less-efficient approach for V1.